format-objc-hook

```diff
@@ -19,18 +19,24 @@ objc_files=$(objc_files_to_format "$1")
 function format_objc() {
   success=0
   for file in $objc_files; do
-	difference=$("$DIR"/format-objc-file-dry-run.sh "$file" | diff "$file" - | wc -l)
+    difference=$("$DIR"/format-objc-file-dry-run.sh "$file" | diff "$file" - | wc -l)
 	
     if [ $difference -gt 0 ]; then
+        if [ $success -eq 0 ]; then
+            echo -e "🚸 Format and stage individual files:"
+        fi
     	# This is what the dev can run to fixup an individual file
-    	echo "\"$DIR\"/format-objc-file.sh '$file' && git add '$file';"
+    	echo -e "\"$DIR\"/format-objc-file.sh '$file' && git add '$file';"
```

Comment:
NIT: I don't think you need the `-e` here, because there is no `\n` on the line.